test/builder/QuarkBuilderTransfer.t.sol

```diff
@@ -424,6 +455,163 @@ contract QuarkBuilderTransferTest is Test, QuarkBuilderTest {
         assertNotEq(result.eip712Data.hashStruct, hex"", "non-empty hashStruct");
     }
 
+    function testSimpleBridgeWithAcrossTransferSucceeds() public {
```

Comment:
I think a better place to put this would be in `BridgeLogic.t.sol` so we have a unified place to find the bridging tests.